nilearn/plotting/surface/tests/test_matplotlib_backend.py

```diff
@@ -1,9 +1,10 @@
+"""Test nilearn.plotting.surface._matplotlib_backend functions."""
+
 # ruff: noqa: ARG001
 
 import re
 import tempfile
 
-import matplotlib.pyplot as plt
```

Comment:
Removed this, instead I use `matplotlib_pyplot` as it is specified as a param to all test functions.